src/pages/Home/Testimonials/index.tsx

```diff
@@ -0,0 +1,66 @@
+import {
+  Badge,
+  Box,
+  Center,
+  Container,
+  Grid,
+  Text,
+  Title,
+} from "@mantine/core";
+import TestimonialBox from "../../../components/TestimonialBox";
+import classes from "./index.module.css";
+
+export default function Testimonials() {
+  const allData = [
```

Comment:
Eggu u should add me in here and pick random 3 to show

Reply:
actually i would pick 3 testers (connect be to fe fast pls)

Reply:
> actually i would pick 3 testers (connect be to fe fast pls)

backlog belike : come here

Reply:
@pdt1806 okey make a place for testers to put their comment on fe to be